cpp/path_module/algorithm/path.cpp

```diff
@@ -4,6 +4,69 @@
 
 #include "mgp.hpp"
 
+void Path::Elements(mgp_list *args, mgp_func_context *ctx, mgp_func_result *res, mgp_memory *memory) {
+  mgp::MemoryDispatcherGuard guard(memory);
+  const auto arguments = mgp::List(args);
+  auto result = mgp::Result(res);
+
+  try {
+    const auto path{arguments[0].ValuePath()};
+    mgp::List split_path(path.Length() * 2 + 1);
```

Comment:
Maybe store path.Length() in a variable to avoid calling it every loop iteration in the below code.